language/move-prover/boogie-backend-v2/src/prelude/multiset-array-theory.bpl

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// Boogie model for multisets, based on Boogie arrays. This theory assumes extensional equality for element types.
+
+type {:datatype} Multiset _;
```

Comment:
I suppose this one is being used for Move events.

Reply:
Yes